src/pages/MovieList.jsx

```diff
@@ -0,0 +1,170 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import '../styles/movielist.css'
+import Magnify from '../../public/search.svg'
+import Ig from '../../public/instagram.svg'
+import Git from '../../public/github.svg'
+import Lk from '../../public/linkedin.svg'
+import Logo from '../../public/logo.png'
+
+function MovieList() {
+  const [list, setList]= useState([])
+  const [page, setPage]= useState(1)
+  const [expand, setExpand]= useState(0)
+  const [clicked, setClicked]= useState(false)
+  const [load, setLoad]= useState(true)
+
+  function getExpanded(id) {
+
+    if(clicked === true) {
+      setClicked(false)
+      setExpand(0)
+    }
+    else {
+      setClicked(true)
+      setExpand(id)
+    }
+  }
+
+  function getData() {
+    axios.get(`https://api.themoviedb.org/4/list/1?page=${page}&api_key=9e89dc1810e00461d6f59011e04175ed`).then((movies) => {
+      setTimeout(() => setLoad(false), 1000)
+      setList(movies.data.results)
+    })
+  }
```

Comment:
I think it's better that you don't hardcode the loading time, you could just use async await and setLoad to false when the get movie is finished.

also, i think you need an error handling here because we don't know if this API call will always succeed or not.